doc/manuals/user/subscriptions_alttype.md

```diff
@@ -1,40 +0,0 @@
-# Subscriptions based in alteration type
```

Comment:
https://github.com/telefonicasc/devicelocation-vertical/pull/1

Reply:
Done in 89be00c229c5cf8afb8245aafc9a0ab077f9a369